packages/view-types/heatmap/src/HeatmapOptions.js

```diff
@@ -47,6 +53,20 @@ export default function HeatmapOptions(props) {
           </OptionSelect>
         </TableCell>
       </TableRow>
+      <TableRow>
+        <TableCell className={classes.labelCell}>
+          Disable Tooltip
```

Comment:
```suggestion
          Tooltips Visible
```

Prefer the user does not have to think about double negatives (e.g., disabled is unchecked -> not not visible vs. visible is checked -> visible) 

Ditto for Spatial/Embedding options component

(for variable names it is less important but at least we should try to stick to this convention for the UI)

Reply:
Good point. I did it in the double negatives way, just so that I am consistent with the already existing prop called `disableTooltip`. I have renamed all but this one to rather use "tooltipVisible". I didn't rename the prop `disableTooltip`, because I wanted to avoid breaking the view-config schema for any users that already are using this prop in their config.